api/src/controllers/auth.ts

```diff
@@ -17,63 +22,65 @@ import logger from '../logger';
 const router = Router();
 
 const loginSchema = Joi.object({
-	email: Joi.string().email().required(),
+	email: Joi.string().required(),
 	password: Joi.string().required(),
 	mode: Joi.string().valid('cookie', 'json'),
 	otp: Joi.string(),
 }).unknown();
```

Comment:
Any ideas on where / how to store these provider-specific body definitions properly?